### PR TITLE
Allow metrics 'crossentropy', 'ce'

### PR DESCRIPTION
**Pull request by ozabluda**
...similar to 'accuracy', 'acc'.

`sparse_categorical_crossentropy: + val_sparse_categorical_crossentropy:` take way too much room in the progbar.

### CODE REVIEW

**Comment by ozabluda** [comment]
`'xE'` would be much cooler

**Comment by fchollet** [comment]
That's a reasonable addition. Maybe we'd need to refactor it to share code with the support for accuracy (and keep the codebase simpler).

I've never seen the abbreviation "ce" for crossentropy. "xent" is more common. But I don't think we should include either.

**Comment by ozabluda** [comment]
"ce" is used about 2x more often than "xent", but we can include both:
(5,480 results) https://www.google.com/search?q="cross+entropy"+"CE"+-method 
(2,400 results) https://www.google.com/search?q="cross+entropy"+"xent" 

"crossentropy" is 10 chars longer than "ce", and it's listed on progbar line twice, for the total win of 20 chars, which is a lot.

I don't see any way to refactor it while making code *simpler*. If there is more of that (which I don't envision), then we can refactor to avoid code triplication while making code *more complex*.

**Comment by ozabluda** [comment]
@fchollet, should we allow 'xe' as well?
